specification/databox/resource-manager/readme.md

```diff
@@ -108,6 +108,15 @@ csharp:
   clear-output-folder: true
 ```
 
+### Shipping address required properties as of 2020-04-01
+``` yaml
```

Comment:
This needs to go in the previous section, so it only applies to sharp generation